pulp-glue/pulp_glue/common/openapi.py

```diff
@@ -504,20 +504,21 @@ def render_request_body(
                 elif content_type.startswith("application/x-www-form-urlencoded"):
                     data = body
                 elif content_type.startswith("multipart/form-data"):
-                    uploads: Dict[str, UploadType] = {}
+                    uploads: Dict[str, Tuple[str, UploadType, str]] = {}
                     data = {}
                     # Extract and prepare the files to upload
                     if body:
                         for key, value in body.items():
                             if isinstance(value, (bytes, BufferedReader)):
-                                uploads[key] = value
+                                uploads[key] = (
+                                    getattr(value, "name", key),
```

Comment:
I believe the python bindings send files with the key as the filename also. But let's do it this way now.
(Actually I always assumed, DRF or at least Pulp would ignore the filename also.)